.vscode/extensions.json

```diff
@@ -0,0 +1,13 @@
+{
+    "recommendations": [
+        "streetsidesoftware.code-spell-checker",
+        "ms-vscode.cpptools",
+        "ms-vscode.cpptools-themes",
+        "ms-vscode.cpptools-extension-pack",
+        "geequlim.godot-tools",
+        "esbenp.prettier-vscode",
+        "ms-python.python",
+        "ms-python.debugpy",
+        "ms-python.vscode-pylance"
```

Comment:
```suggestion
        "esbenp.prettier-vscode"
```
I wouldn't add python since we don't have python

Reply:
Or is it helping GDScript? I've been working with GDScript in VScode with just godot-tools and haven't had any speedbumps